src/main/java/org/jabref/gui/entryeditor/EntryEditor.java

```diff
@@ -1148,7 +1148,7 @@ public void stateChanged(ChangeEvent event) {
                 }
 
                 // When the tab "Related articles" gets selected, the request to get the recommendations is started.
-                if (activeTab instanceof EntryEditorTabRelatedArticles) {
+                if (activeTab == relatedArticlePanel) {
```

Comment:
Hm. I really wonder why this previously didn't work. But well, if it works now, then it's fine...